viz.yaml

```diff
@@ -302,4 +295,18 @@ publish:
       resources: ["lib_d3_js", "topojson", 
                   "map_css", "custom_css", "hashes", "projection", "map_utils", 
                   "map_counties", "legend_utils", "pie_utils"]
-      embed: ["map", "text_section"]
+      embed: ["map","directions_text"]
+  -
+    id: description_text
```

Comment:
ah, now i see the beauty of your first approach of defining a new template - i did like the consolidation into one text_section item. but even so i think this most recent change/reversion is still the way to go to support future vizzies.

Reply:
And we can still do that approach, right?

Reply:
oh! i guess i don't understand the system all that well, because i thought we must need `context : text` to be named just like that to work with the `printall` template. keeping consolidated would be excellent - how would it work?

Reply:
Maybe I'm not really following this conversation correctly. I think anytime you can define your own template in an individual vizlab project. And now as a bonus, if you wrap it in `<div class="viz-text">`, you'll get Marty's fancy style. But, if you just want to push a little text in like we've done it historically in water-use (using the vizlab printAll template), then you still get the nice style. Anywho, I approve this and the other pull request.